src/app/base/components/MainContentSection/MainContentSection.test.tsx

```diff
@@ -22,9 +22,10 @@ it("can render without a header", () => {
   renderWithMockStore(
     <MainContentSection header={null}>content</MainContentSection>
   );
-  expect(
-    screen.queryByRole("banner", { name: "main content" })
-  ).not.toBeInTheDocument();
+
+  // ContentSection.Header doesn't have an accessible role, and also
+  // does not accept aria labels or test ids in its props.
+  expect(document.querySelector(".content-section__header")).toBeNull();
```

Comment:
Normally I wouldn't do this, but as the comment explains, it's not currently possible to query for this element using testing-library methods. 

A fix for this could be to alter `ContentSection.Header` to use a `header` element as opposed to a `div` - this would give it an accessible role of "banner", and would make it possible to query for using testing-library.